hack/generate/samples/internal/go/v3/memcached_with_webhooks.go

```diff
@@ -437,6 +437,7 @@ func (r *MemcachedReconciler) deploymentForMemcached(m *cachev1alpha1.Memcached)
 									"ALL",
 								},
 							},
+							RunAsUser: &[]int64{1000}[0],
```

Comment:
```suggestion
							// The Memcached requires an user to run
							// Note that for a container be restried this user cannot be 0. 
							// More info: https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted
							RunAsUser: &[]int64{1000}[0],
```

Reply:
Added a comment in https://github.com/operator-framework/operator-sdk/pull/5904/commits/91164a2b327997445b9449eabb9b19a604bd3351 that should resolve this.